src/options/options.css

```diff
@@ -20,51 +20,51 @@ body.mobile .mobile-incompatible {
   opacity: 0.4;
 }
 
-ul {
-  margin: 0px;
-  padding: 0px;
-}
-li {
-  list-style-type: none;
-  margin-top: 10px;
-  padding: 0px;
+form {
+  margin: 16px 32px;
 }
-/* in a fieldset e.g. we use a condensed version */
-.condensed-list {
-  margin-left: 0px;
+
+h1 {
+  margin: 0;
+  padding: 16px 0;
+
+  border-top: 1px solid rgba(12, 12, 13, 0.15);
+  font-weight: 300;
+  font-size: 1.46em;
```

Comment:
What makes you choose this font-size? [The photon guide](https://design.firefox.com/photon/visuals/typography.html#scale) lists the [title](https://design.firefox.com/photon/visuals/typography.html#title) as "Title 30", which seems to be `22px`.

Reply:
oh, i dug into the actual css. okay, i will follow this instead

Reply:
Yes, it may be actualyl result in the same value in calculations. Possibly... 😉 But better to use the documented values. 😄